src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+  
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+  
+.stars--0 :nth-child(-n-0),
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

```suggestion
  background-image: url(images/star-active.svg);
```
There is no need to duplicate background-repeat property